### ISSUE TITLE
check existing kaggle models

### ISSUE THREAD

**Issue by kaichop** [WGLab/Project_Belka#4]
Some people have shared their code using xgboost and random foreset for the prediction. We can borrow the code both for data processing and for prediction, so that we do not need to start from scratch. Compile the information in this issue.

This is a pinned example https://www.kaggle.com/code/andrewdblevins/leash-tutorial-ecfps-and-random-forest that we can reproduce and learn how to use parquet to process the data, and how to use 42 as random seed to ensure consistency of training/testing using different models in the future.

**Comment by wangwpi** [comment]
I'm going to check the **pinned example (random forest)** and the **BERT-fine tuning** model on the kaggle.

**Comment by wangwpi** [comment]
The pinned example (random forest) has been reproduced and the public score for that model is 0.263. The jupyter notebook has been uploaded in the `models/Leash_Tutorial_test.ipynb` in this repository.

**Comment by kaichop** [comment]
is it affected by "LIMIT 30000" in the SQL code?

**Comment by wangwpi** [comment]
Yes you are right, in the tutorial the model was trained only on
30000+30000 samples, I will try to train using the whole training dataset
and see the performance.

**Comment by wangwpi** [comment]
I have uoloaded my notebook for BERT fine tunning (use 60000 data), and a current Neural Network Model using all split data (230M training, 56M validation). The morgan fingerprint for all split data are generated in trunks (500K each trunk) as numpy array file.
